data/_ui/tooltips.txt

```diff
@@ -1463,3 +1463,9 @@ tip "Scroll speed"
 
 tip "Date format"
 	`The format that dates in the game are displayed in.`
+
+
+
+# Player info panel
+tip "crew column"
+	`Crew currently aboard the ship. May be fewer than minimum complement.`
```

Comment:
```suggestion
	`Crew currently aboard the ship. This number may be fewer than minimum complement.`
```
What about this?

Reply:
I would have either
> May be fewer than minimum complement.
or
> This number may be fewer than _the_ minimum complement.
but my preference would be for brevity. this is a treansient tooltip not prose.

Reply:
Forgot the the, correct.